### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by edmundnoble** [scalaz/blog#1]
GPLv3 for the build code, any license the author wants for the posts?

**Comment by emilypi** [comment]
I'd be perfectly fine for authors maintaining intellectual property rights to their own blog posts, however, it would be a potential mess if people are allowed to revoke/change/otherwise edit their posts with total authority. I would suggest an authorship credit go to the original author, with all rights to republish elsewhere granted, but the right to revoke/change/otherwise edit be subject to a Scalaz community democratic vote (e.g. submitting a pull request for changes).

I'm not sure if there's a license which perfectly details that kind of scheme.

**Comment by jmcardon** [comment]
:+1:

**Comment by hrhino** [comment]
Can't we just do CC-BY-SA? That would give "us" (scalaz) the right to publish it, but it doesn't give the author the ability to demand that we take it down or modify it.

**Comment by jdegoes** [comment]
Authors should have full rights over their posts. It's enough they give a non-exclusive, worldwide, royalty free, perpetual license to distribute the content on the blog. 

If some authors want to do CC-BY-SA or Beerware, then good on them, but let's not make this religious. People who create content can do whatever they damn well like with it.

**Comment by edmundnoble** [comment]
Yeah, I tend to agree with @jdegoes here. It isn't too much of a hassle to give authors full rights, IMO. They can license whatever they want.
